### DISCUSSION TITLE
Update From 5.0.0 to 5.1.1 - Missing operation-scope entries in DB, can't see tables, curious on PHP worker.

### DISCUSSION THREAD

**Discussion by cryoaura** [apioo/fusio#565] [Q&A] [answered]
Hey there!

When updating to 5.1.1, i had the same issue as my move from 4.x to 5.0.0 - the new features didn't add the operation-scope links into the DB. I followed the same idea as last time when you assisted with the token and webhook features and it allowed me access to the database section in developer. Needed to add all the operations to the backend.database scope.

That being said, I didn't see anything in the migrate command. No errors or anything. Said no changes were needed.

Now with the database section, it says to pick your connection - there's a SYSTEM option. I choose that, but no tables show in the list. Am I supposed to create a new connection to the Fusio DB or was the intent that there should be one by default?

I also noticed the notes said something about a built-in local PHP worker now - Is this a connection that should be there? I'm fine with the workers being separate, but thought it was a neat idea that you were adding a local php worker.

Loving the progress and features that are being added!

**Comment by chriskapp** [comment]
@cryoaura ok thanks for the info and feedback, normally we have a method which should automatically sync and add scopes of new operations s. https://github.com/apioo/fusio-impl/blob/master/src/Migrations/Version20240712210005.php#L41 but it looks like this works not properly, I will try to check this for the next release.

> Now with the database section, it says to pick your connection - there's a SYSTEM option. I choose that, but no tables show in the list. Am I supposed to create a new connection to the Fusio DB or was the intent that there should be one by default?

Yes indeed the intent is that all Fusio tables are excluded from the database panel, the system connection has a filter to exclude those tables, so that you can start to build your API on an clean database. But it is always possible to create a new connection to the same database which then would include the Fusio tables since the filter exists only on the system connection, but in general there should be no need to modify the Fusio tables.

The PHP Worker Local is basically a new action which does not need a worker connection since the code gets executed on the same server, so you can use it directly without configuration.

**Reply by chriskapp** [reply, marked as answer]
FYI I have just fixed s. https://github.com/apioo/fusio/issues/566 so in the future new scopes should be automatically added as expected

**Reply by cryoaura** [reply]
Awesome! I just checked out the local php worker - at first I thought it would be helpful, then thought about it, and absolutely love having the ability to have multiple php workers as connections since you added the features.

As usual, appreciate the help! Fusio is starting to feel more and more feature-rich and polished each week!

**Reply by cryoaura** [reply]
Also forgot to mention I see the purpose behind the database section now and love it. We have a staging DB server just because it was easier to manage things on a MS SQL server in our environment. We may be able to abolish that if this is easy to maintain our staging data which would be cool!
